app/models/annotation/AnnotationSettings.scala

```diff
@@ -1,12 +1,19 @@
 package models.annotation
 
+import com.scalableminds.util.enumeration.ExtendedEnumeration
 import com.scalableminds.webknossos.tracingstore.tracings.TracingType
+import com.scalableminds.webknossos.tracingstore.tracings.TracingType.TracingType
 import com.scalableminds.webknossos.tracingstore.tracings.volume.ResolutionRestrictions
-import models.annotation.AnnotationSettings._
+import models.annotation.AnnotationSettings.skeletonModes
 import play.api.libs.json._
 
+object TracingMode extends ExtendedEnumeration {
+  type TracingMode = Value
+  val orthogonal, oblique, flight, volume = Value
+}
+
 case class AnnotationSettings(
-    allowedModes: List[String] = SKELETON_MODES,
+    allowedModes: List[TracingMode.Value] = skeletonModes,
```

Comment:
Note for reviewer: in the files where the enums are defined, their respective type aliases cannot be imported (that causes conflicts), hence this is still `TracingMode.Value` here, but would be `TracingMode` elsewhere.